astropy/visualization/lupton_rgb.py

```diff
@@ -211,10 +226,17 @@ def __init__(self, minimum=None, maximum=None, image=None):
                 raise ValueError("minimum and maximum values must not be equal")
             self._range = float(maximum - minimum)
 
+    # copies -- could work row by row to minimise memory usage
```

Comment:
Is this a newly-added comment?

Reply:
Oops this is a stray comment I inserted by accident when refactoring. I'll remove it